src/time/span.cr

```diff
@@ -53,13 +53,15 @@ struct Time::Span
   # @nanoseconds can either be negative or positive).
   @nanoseconds : Int32
 
-  def self.new(hours : Int, minutes : Int, seconds : Int)
+  @[Deprecated("Use `new` with named arguments instead.")]
+  def self.new(_hours : Int, _minutes : Int, _seconds : Int)
     new(0, hours, minutes, seconds)
```

Comment:
This ain't compile anymoar.

Reply:
What do you mean?

Reply:
`_hours` vs `hours` etc

Reply:
We are still missing to fix this `_arg` vs `arg`.
It's fine to leave a at least a typeof spec to ensure it compiles. Or even that it still behaves as wanted. Warnings are allowed in the CI.